app/bundles/LeadBundle/Form/Type/LeadCategoryType.php

```diff
@@ -22,7 +22,7 @@ public function configureOptions(OptionsResolver $resolver): void
     {
         $resolver->setDefaults([
             'choices'           => function (Options $options): array {
-                $categories = $this->categoryModel->getLookupResults('global', '', 300);
+                $categories = $this->categoryModel->getLookupResults('email', '', 0);
```

Comment:
Why email and 0?

Reply:
> Why email 
I am unable to recall, but I will update that with Global. 
> and 0
0 is passed to remove any limit.

Here, I'll keep the original version rather than this updated one.